tests/chat_test.py

```diff
@@ -0,0 +1,76 @@
+import asyncio
+import os
+import shutil
+import subprocess
+import tempfile
+import time
+
+from pykeybasebot import Bot, kbsubmit
+from pykeybasebot.types import chat1
+
+import pytest
+import yaml
+
+
+# create temp dir
+def create_working_dir():
+    with tempfile.TemporaryDirectory() as tmpdir:
+        kb_location = shutil.which("keybase").strip()
+        kb_destination = f"{tmpdir}/keybase"
+        shutil.copy(kb_location, kb_destination)
+        print(os.listdir(tmpdir))
+
+
+@pytest.fixture()
+def config():
+    with open("tests/test_config.yaml") as f:
+        data = yaml.safe_load(f)
+        return data
+
+
+@pytest.fixture()
+def channel(config):
+    return chat1.ChatChannel(
+        name=f"{config['bots']['alice']['username']},{config['bots']['bob']['username']}"
+    )
+
+
+@pytest.fixture()
+async def bot(config):
+    with tempfile.TemporaryDirectory() as tmpdir:
+        kb_location = shutil.which("keybase").strip()
+        kb_destination = f"{tmpdir}/keybase"
+        shutil.copy(kb_location, kb_destination)
+
+        args = [kb_destination, "--home", tmpdir, "service"]
+        subprocess.Popen(args)
+        time.sleep(2)
+
+        bot = Bot(
+            username=config["bots"]["alice"]["username"],
+            paperkey=config["bots"]["alice"]["paperkey"],
+            handler=noop_handler,
+            home_path=tmpdir,
+            keybase=kb_destination,
+        )
+        yield bot
+        await bot.teardown()
+        await kbsubmit(kb_destination, "ctl stop")
+
+
+def noop_handler(*args, **kwargs):
+    pass
+
+
+@pytest.mark.asyncio
+async def test_list(bot):
+    conversations = await bot.chat.list()
+    for conversation in conversations:
+        assert type(conversation) is chat1.ConvSummary
```

Comment:
Anything else I should test in the results? I want to keep them general enough to pass on any test account, but also specific enough to be meaningful.